api/src/application/user/user-controller.js

```diff
@@ -53,9 +53,12 @@ module.exports = (
           await userEmails.sendChangeEmailVerifyEmail(user, client, email, request.query);
           break;
         case 'cancel_new':
+          const emailToken = bookshelf.model('email_token').where({user_id:user.get('id')})
```

Comment:
I think having the query defined up here and the `.destroy()` call later is a bit confusing.

and actually just doing this without defining a variable is ok (and actually what I would have done):

```
await bookshelf.model('email_token')
  .where({user_id:user.get('id')})
  .destroy()
```